zerver/openapi/zulip.yaml

```diff
@@ -7630,6 +7630,11 @@ paths:
         settings for existing users in any way.
 
         **Changes**: New in Zulip 5.0 (feature level 96).
+        Endpoint added to update default values of user settings in realm.
+
+        **Changes**: New in Zulip 5.0 (feature level 99).
+        Previously, `twenty_four_hour_time` parameter was edited using the
+        `default_twenty_four_hour_time` parameter to the `PATCH /realm` endpoint.
```

Comment:
I think we generally want the newest change to be on top, so these should be swapped.  

Also, the "**Changes**: New in Zulip 5.0 (feature level 96)." line is meant to stand on its own and say that the endpoint (or parameter) itself is new in that version.  It's possible we should rethink that wording, since your edit here suggests that wasn't obvious to you, but I'd expect the "feature level 99" line to be more like "Changes: Added `twenty_four_hour_time` parameter" or something like it.

(Though it's also possible we should just have that one be an exception to the sharing approach taken here).

Reply:
Since both changes are in Zulip 5.0, perhaps we could combine the two into one statement? I saw a few examples as I was skimming through the other change notes in the documentation. I'm thinking something like ...

> **Changes**: New in Zulip 5.0 (feature level 96). `twenty_four_hour_time` parameter added in Zulip 5.0 (feature level 99), which was previously edited using `default_twenty_four_hour_time` parameter to the `PATCH /realm` endpoint.